public/app/features/auth-config/utils/data.ts

```diff
@@ -56,7 +56,9 @@ const strToValue = (val: string | string[]): SelectableValue[] => {
   }
   // Stored as JSON Array
   if (val.startsWith('[') && val.endsWith(']')) {
-    return JSON.parse(val).map((v: string) => ({ label: v, value: v }));
+    try {
```

Comment:
Could you please add a comment just to clarify why the try-catch is necessary here? Thanks!